nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/CheckInDocument.java

```diff
@@ -54,7 +59,7 @@
     protected String versionVarName;
 
     protected VersioningOption getVersioningOption() {
-        return "major".equalsIgnoreCase(version) ? VersioningOption.MAJOR : VersioningOption.MINOR;
+        return options.getOrDefault(version, VersioningOption.MINOR);
```

Comment:
That's a change of semantics, it doesn't ignore the case anymore.

Reply:
then `return options.getOrDefault(version.toLowerCase(), VersioningOption.MINOR);`